ghcup

```diff
@@ -453,6 +453,26 @@ edo()
     "$@" || exit 2
 }
 
+# @FUNCTION: emake
+# @USAGE: [arguments]
+# @DESCRIPTION:
+# Wrapper around 'make', may call 'gmake' if it exists.
+emake() { # avoid re-checking for gmake
+    if [ -n "${MAKE}" ] ; then
+        edo "${MAKE}" "$@"
```

Comment:
You are still quoting `${MAKE}`, which we shouldn't in this case. The variable assignment can be quoted, that doesn't matter, but not the invocation. Use `# shellcheck disable=SC2086` on those edo lines to silence the warning.

Reply:
Done